src/content/tutorials/fleet-management/cluster-management/migration-to-cluster-api/_index.md

```diff
@@ -55,3 +55,181 @@ Our engineers will check that all resources and infrastructure are correctly mig
+-            irsaAdditionalDomain: irsa.foo.k8s.bar.acme.net
+     cluster:
+         internal:
+             advancedConfiguration:
+@@ -32,43 +29,23 @@
+                         etcdPrefix: giantswarm.io
+                         extraCertificateSANs:
+                             - api.foo.k8s.bar.acme.net
+-                    etcd:
+-                        initialClusterState: existing
+-                        experimental:
+-                            peerSkipClientSanVerification: true
+                     files:
+-                        - contentFrom:
+-                            secret:
+-                                name: foo-migration-custom-files
+-                                key: join-etcd-cluster
+-                          path: /migration/join-existing-cluster.sh
+-                          permissions: "0644"
+-                        - contentFrom:
+-                            secret:
+-                                name: foo-migration-custom-files
+-                                key: move-etcd-leader
+-                          path: /migration/move-etcd-leader.sh
+-                          permissions: "0644"
+-                        - contentFrom:
+-                            secret:
+-                                name: foo-migration-custom-files
+-                                key: api-healthz-vintage-pod
+-                          path: /etc/kubernetes/manifests/api-healthz-vintage-pod.yaml
+-                          permissions: "0644"
+                         - contentFrom:
+                             secret:
+                                 name: foo-migration-custom-files
+                                 key: add-extra-service-account-key
+                           path: /migration/add-vintage-service-account-key.sh
+                           permissions: "0644"
+                           - contentFrom:
+                            secret:
+                                name: foo-sa-old
+                                key: tls.key
+                          path: /etc/kubernetes/pki/sa-old.pem
+                          permissions: "0640"
+                     preKubeadmCommands:
+-                        - 'iptables -A PREROUTING -t nat  -p tcp --dport 6443 -j REDIRECT --to-port 443 # route traffic from 6443 to 443'
+-                        - 'iptables -t nat -A OUTPUT -p tcp --destination 127.0.0.1 --dport 6443 -j REDIRECT --to-port 443 # include localhost'
+-                        - /bin/sh /migration/join-existing-cluster.sh
+                         - /bin/sh /migration/add-vintage-service-account-key.sh
+-                        - sleep 90
+-                    postKubeadmCommands:
+-                        - /bin/sh /migration/move-etcd-leader.sh
++        providerIntegration:
++            controlPlane:
++                kubeadmConfig:
++                    clusterConfiguration:
++                        apiServer:
++                            serviceAccountIssuers:
```

Comment:
Should we document right now what the order here means? And how/when the customer can remove this? Fine to leave it for another ticket if you think it's out of scope.

Reply:
I've already picked up the task to document that as well, but I can do it in another PR 👌